PURL-SPECIFICATION.rst

```diff
@@ -206,44 +213,61 @@ The rules for each component are:
+
     - MUST NOT contain a '/'
     - MUST NOT be any of '..' or '.'
     - MUST NOT be empty
+
   - The ``subpath`` MUST be interpreted as relative to the root of the package
 
 
 Character encoding
 ~~~~~~~~~~~~~~~~~~
 
-For clarity and simplicity a ``purl`` is always an ASCII string. To ensure that
+For clarity and simplicity, a ``purl`` is always an ASCII string. To ensure that
 there is no ambiguity when parsing a ``purl``, separator characters and non-ASCII
-characters must be UTF-encoded and then percent-encoded as defined at::
+characters MUST be UTF-encoded and then percent-encoded as defined at
+https://en.wikipedia.org/wiki/Percent-encoding and as further defined below.
+
+Use these rules for percent-encoding and decoding ``purl`` components.  Except
+as otherwise provided in the "Rules for each ``purl`` component" section above:
+
+- A character used in a ``purl`` component MUST be percent-encoded unless it is:
 
-    https://en.wikipedia.org/wiki/Percent-encoding
+  (1) an unreserved character as defined in RFC 3986 section 2.3 (https://datatracker.ietf.org/doc/html/rfc3986#section-2.3),
 
-Use these rules for percent-encoding and decoding ``purl`` components:
+  (2) expressly defined in this PURL-SPECIFICATION.rst as a ``purl`` separator (and only when used as such a separator), or
 
-- the ``type`` must NOT be encoded and must NOT contain separators
+  (3) expressly permitted in that ``purl`` component.
 
-- the '#', '?', '@' and ':' characters must NOT be encoded when used as
-  separators. They may need to be encoded elsewhere
+- All non-ASCII characters MUST be encoded as UTF-8 and then percent-encoded.
 
-- the ':' ``scheme`` and ``type`` separator does not need to and must NOT be encoded.
-  It is unambiguous unencoded everywhere
+- The characters used as ``purl`` separators are listed below.  These characters:
 
-- the '/' used as ``type``/``namespace``/``name`` and ``subpath`` segments separator
-  does not need to and must NOT be percent-encoded. It is unambiguous unencoded
-  everywhere
+  - MUST NOT be percent-encoded when used as separators.
+  - MUST be percent-encoded when not used as separators unless expressly permitted
+    by a ``purl`` component.
 
-- the '@' ``version`` separator must be encoded as ``%40`` elsewhere
-- the '?' ``qualifiers`` separator must be encoded as ``%3F`` elsewhere
-- the '=' ``qualifiers`` key/value separator must NOT be encoded
-- the '#' ``subpath`` separator must be encoded as ``%23`` elsewhere
+.. list-table::
+   :widths: 1 4
+   :header-rows: 1
 
-- All non-ASCII characters must be encoded as UTF-8 and then percent-encoded
+   * - Character
+     - Use as separator
+   * - ':'
+     - between ``scheme`` and ``type``
+   * - '@'
+     - ``version`` prefix
+   * - '?'
+     - ``qualifiers`` prefix
+   * - '#'
+     - ``subpath`` prefix
+   * - '/'
+     - ``type``/``namespace``/``name`` and ``subpath`` segments separator
+   * - '='
+     - ``qualifiers`` ``key``/``value`` separator
```

Comment:
add  
```rst
   * - '&'
     - ``qualifiers`` ``key=value`` separator
```

Reply:
Thanks for catching this @jkowalleck -- :+1:  I've added it to the update of this PR that I'll be pushing shortly.